src/content/reference/cli.mdx

```diff
@@ -604,6 +604,22 @@ You need to be [logged in](reference/cli.mdx#context) to Okteto before running t
 | :-------- | :----: | :------------------------------------------------------------------------------------- |
 | _--label_ | (list) | tag and organize preview environments using labels (multiple _--label_ flags accepted) |
 
+#### sleep
+
+Sleeps an okteto preview environment. You need to provide that preview name as an argument to the command.
+
+```console
+$ okteto preview sleep [name]
+```
+
```

Comment:
Is there any permission restriction worth mentioning? For example can anyone set to sleep global preview environments?

Reply:
only admins should be able to do that action (or the owner of the preview env)

Reply:
let's mention it then, wdyt @teresaromero?

Reply:
I think so, I was thinking something simple like we have for `deploy`, search for: "Only users with admin access can deploy a global preview environment."